components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationMgtDBQueries.java

```diff
@@ -446,4 +446,9 @@ public class ApplicationMgtDBQueries {
 
     public static final String GET_TOTAL_SP_CLAIM_USAGES = "SELECT COUNT(*) FROM SP_CLAIM_MAPPING WHERE TENANT_ID = ?" +
             " AND IDP_CLAIM = ?";
+
+    public static final String GET_APPLICATION_ROLES = "SELECT ROLE_NAME FROM SP_APP_ROLE WHERE SP_APP_ID = ?";
```

Comment:
Is this a new table? have you included the DDLs? also what is the migration impact?